terraform/ecs_asg/iam.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_iam_role_policy" "publish_to_sns" {
+  name   = "publish_to_sns"
+  role   = "${aws_iam_role.role.id}"
+  policy = "${var.publish_to_sns_policy}"
+}
```

Comment:
It's only used there. It's to give permission the tole declared below to publish to use the policy passed in in the variables